update-translations.py

```diff
@@ -57,6 +57,9 @@ def remove_current_translations():
     '''
     for (_,name) in all_ts_files():
         os.remove(name)
+    remove_orig_files()  # TODO interim, remove eventually
```

Comment:
Agree with @laanwj -- no need to call `remove_orig_files()` here. Even any `*.orig` files left (for instance, due to an interrupted script execution), on Unix systems the following `os.rename()` call will replace destination file [silently](https://docs.python.org/3/library/os.html#os.rename).